src/applications/disability-benefits/all-claims/components/ComboBox.jsx

```diff
@@ -233,6 +239,25 @@ export class ComboBox extends React.Component {
     }
   };
 
+  setSelectedState(option) {
```

Comment:
Should this also be used in the `handleClickOutsideList` method? I believe the behavior for clicking outside the list should be the same as for tab and escape.  When I was testing with voiceover and I entered a search term and then clicked outside the list it would not announce what was selected like tab and escape do.

Reply:
Good catch! Most likely a screenreader user would be triggering the keyboard version, but sometimes they switch back and forth between modalities. I'll add that in